gnomad_mitochondria/resources/init-vep101.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+export PROJECT="$(gcloud config get-value project)"
+export VEP_CONFIG_PATH="$(/usr/share/google/get_metadata_value attributes/VEP_CONFIG_PATH)"
+export VEP_REPLICATE="$(/usr/share/google/get_metadata_value attributes/VEP_REPLICATE)"
+export VEP_BUCKET=hail-${VEP_REPLICATE}-vep
+export ASSEMBLY=GRCh38
+export VEP_DOCKER_IMAGE=gcr.io/broad-mpg-gnomad/vep_101
+
+mkdir -p /vep_data/loftee_data
+mkdir -p /vep_data/homo_sapiens
+
+PACKAGES="slackclient==2.9.3 sklearn tabulate scipy statsmodels ggplot hdbscan websocket-client scikit-learn joblib"
+pip install --upgrade --no-deps $PACKAGES
```

Comment:
This seems rather out of place since it's not related to VEP. I assume these packages are here because they are dependencies of `gnomad`? If so, I think this could be replaced with passing `--packages gnomad` to `hailctl dataproc start`.

Reply:
would --packages gnomad mean using a  less recent version of the code? I think I could also just cut this back to `pip install --upgrade --no-deps slackclient==2.9.3`

Reply:
`--packages gnomad` would install the latest version on PyPI.

To get the latest version from GitHub, I think you can do `--packages "git+https://github.com/broadinstitute/gnomad_methods.git@master"`.